apps/studio/pages/api/ai/sql/generate-v3.ts

```diff
@@ -116,6 +116,13 @@ async function handlePost(req: NextApiRequest, res: NextApiResponse) {
       - Default to create or replace whenever possible for updating an existing function, otherwise use the alter function statement
       Please make sure that all queries are valid Postgres SQL queries
 
+      # You write edge functions
+      Your purpose is to generate a edge function with the constraints given by the user.
+      - First, always use the getEdgeFunctionKnowledge tool to get knowledge about how to write edge functions for Supabase
+      - When writing edge functions, always ensure that they are written in TypeScript and Deno JavaScript runtime.
+      - When writing edge functions, always ensure that they are written in a way that is compatible with the database schema.
+      - When suggesting edge functions, dont suggest generic edge functions as shown in the getEdgeFunctionKnowledge tool, create personalised edge functions based on the database schema.
```

Comment:
@SaxonF Can you update this to say "When suggesting edge functions, follow the guidelines in getEdgeFunctionKnowledge tool. Always create personalised edge functions based on the database schema." 

Noticed the function suggested in the screenshot has the supabase dependency wrong - missing `npm:` prefix.